query/src/main/scala/filodb/query/LogicalPlan.scala

```diff
@@ -30,7 +30,6 @@ sealed trait LogicalPlan {
   * not in the same time cadence as user query windowing.
   */
 sealed trait RawSeriesLikePlan extends LogicalPlan {
-  override def isRoutable: Boolean = false
```

Comment:
which plans are routable now ? Is this method needed anymore ?

Reply:
Yes we don't want to route ScalarFixedDouble and ScalarTimeBased plans